docs/source/install.rst

```diff
@@ -56,9 +56,27 @@ And then you'll need to install dependencies::
     $ pip install -r dev-requirements.txt
 
 *pythondotorg* will look for a PostgreSQL database named ``pythondotorg`` by
-default. Run the following command to create a new database::
+default. Run the following commands to create a new database.
 
-    $ createdb pythondotorg -E utf-8 -l en_US.UTF-8
+Change to postgres user to perform postgres database administrative tasks::
+
+    $ sudo su - postgres
```

Comment:
Out of curiosity, did you get an error when you ran `createdb pythondotorg -E utf-8 -l en_US.UTF-8`? If you need to be "postgres" user, perhaps passing `-O postgres` might work. Could you please try the following command?

```sh
$ createdb pythondotorg -O postgres -E utf-8 -l en_US.UTF-8
```

Reply:
Indeed i get an error:
createdb: database creation failed: ERROR:  permission denied to create database
Even using createdb pythondotorg -O postgres -E utf-8 -l en_US.UTF-8
I have to log into posgres user to be able to create a database.

Reply:
Thanks for the info. I will make another review tonight, sorry for the delay!

Reply:
I think this can be done a little shorter with the following command:

```sh
$ sudo -u postgres createdb pythondotorg -E utf-8 -l en_US.UTF-8
```

Can you try this command?